{"uid":"6cdaaa15ade4bd92","name":"test_01_phone_login","fullName":"tests.test_00_login.TestLogin#test_01_phone_login","historyId":"6bbda9dd678259dc795c231e4fb7cb31","time":{"start":1656052486122,"stop":1656052486122,"duration":0},"description":"登录账号","descriptionHtml":"<p>登录账号</p>\n","status":"broken","statusMessage":"Exception: activate_app() missing 1 required positional argument: 'self'","statusTrace":"args = (), kwargs = {}\n\n    @wraps(func)\n    def wrapper(*args, **kwargs):\n        try:\n            log.i('--> %s' % func.__qualname__)\n>           return func(*args, **kwargs)\nE           TypeError: activate_app() missing 1 required positional argument: 'self'\n\ncommon/decorator.py:83: TypeError\n\nDuring handling of the above exception, another exception occurred:\n\ncls = <class 'test_00_login.TestLogin'>\n\n    @classmethod\n    def setup_class(cls):\n>       init.activate_app()\n\ntests/test_00_login.py:16: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nargs = (), kwargs = {}\n\n    @wraps(func)\n    def wrapper(*args, **kwargs):\n        try:\n            log.i('--> %s' % func.__qualname__)\n            return func(*args, **kwargs)\n        except requests.ConnectionError:\n            print(\"ConnectionError\")\n            reset_driver()\n            func(*args, **kwargs)\n        except Exception as e:\n            log.e('\\t<-- %s, %s, %s', func.__qualname__, 'Exception', e)\n>           raise Exception(e)\nE           Exception: activate_app() missing 1 required positional argument: 'self'\n\ncommon/decorator.py:90: Exception","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"xunit_setup_class_fixture_TestLogin","time":{"start":1656052484514,"stop":1656052486043,"duration":1529},"status":"broken","statusMessage":"Exception: activate_app() missing 1 required positional argument: 'self'\n","statusTrace":"  File \"/Users/jike/PycharmProjects/pythonProject/venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/jike/PycharmProjects/pythonProject/venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/jike/PycharmProjects/pythonProject/venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/jike/PycharmProjects/pythonProject/venv/lib/python3.9/site-packages/_pytest/python.py\", line 811, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/Users/jike/PycharmProjects/pythonProject/venv/lib/python3.9/site-packages/_pytest/python.py\", line 739, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/jike/PycharmProjects/pythonProject/tests/test_00_login.py\", line 16, in setup_class\n    init.activate_app()\n  File \"/Users/jike/PycharmProjects/pythonProject/common/decorator.py\", line 90, in wrapper\n    raise Exception(e)\n","steps":[],"attachments":[{"uid":"e76410d11b8b7349","name":"init.activate_app-20220624143445.PNG","source":"e76410d11b8b7349.png","type":"image/png","size":554182}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true}],"afterStages":[],"labels":[{"name":"feature","value":"登录"},{"name":"tag","value":"jw"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_00_login"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"jikedeMacBook-Pro-4.local"},{"name":"thread","value":"27175-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_00_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":["jw"]},"source":"6cdaaa15ade4bd92.json","parameterValues":[]}